src/components/QuoteBubble/QuoteBubble.js

```diff
@@ -1,31 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { SPEECH } from '../../constants/speechBubbleVariant';
+import { speechBubbleVariant } from '../../types';
 import { SpeechBubble } from '../Icons';
 import CopyButton from './CopyButton';
 import Wrapper from './Wrapper';
 import Quote from './Quote';
 
+const DEFAULT_QUOTE = "Daily in 5 minutes and I'm still not sure what tot say...";
+
 function copyTextFromBubble(text) {
   navigator.clipboard.writeText(text);
 }
 
-function QuoteBubble({ quote }) {
+function QuoteBubble({ quote = DEFAULT_QUOTE, variant = SPEECH, noCopyToClipboard = false }) {
```

Comment:
`noCopyToClipboard` -> `allowCopy`?

Reply:
I think it better describes what is the real intention here aka hiding copy to clipboard button.

Reply:
So go with `disableCopy`. Anyway you still need to negate that later, so `allowCopy` with default parameter is obvious too.